.github/workflows/ember-compare-asset-sizes.yml

```diff
@@ -0,0 +1,32 @@
+name: Ember compare asset sizes
+
+on:
+  workflow_call:
+    secrets:
+      npm-token:
+        required: true
+      ssh-private-key:
+        required: true
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup npm token for private packages
+      run: echo //registry.npmjs.org/:_authToken=${{ secrets.npm-token }} > ~/.npmrc
+    - name: Setup SSH for Github Auth
+      uses: webfactory/ssh-agent@v0.5.4
+      with:
+        ssh-private-key: ${{ secrets.ssh-private-key }}
+    - uses: actions/checkout@v2.3.4
+      with:
+        # when using pull_request_target, need to specify what sha to compare against (otherwise it does PR base)
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - uses: simplabs/ember-asset-size-action@v2
```

Comment:
I think we are using a fork that Tim made in the student apps at present - https://github.com/blake-education/fast-phonics-client/blob/develop/.github/workflows/compare_ember_asset_sizes.yml#L26 

That's a fork from v1 though - what does v2 add?

Reply:
That particular repo has a fairly unresponsive maintainer, and a mess of forks/branches so we've often had to run off of forks for various features

(For example, DUI is running `uses: fabscale/ember-asset-size-action@fn/handle-chunks` in order to report chunk sizes for ember-auto-import 2)

I think Tim's fork was originally off v1, but looking at it now it looks fairly updated so he might have rebased it or something (probably to get the comment editing feature I added that the maintainer graciously merged)

I'll look a little deeper, but I'll probably update this to Tim's fork to get that show totals stuff.

Reply:
done